plugins/oulogin.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: oulogin
+spec:
+  version: "v0.0.3"
+  homepage: https://github.com/TremoloSecurity/kubectl-login
+  shortDescription: "login to a cluster via OpenUnison from the command line"
```

Comment:
please capitalize.
no need to say "from command line" as that's what plugins are.